components/Main/ClientChatPage.tsx

```
@@ -237,34 +253,43 @@ export default function ClientChatPage({
-                    key={`btn-${nextMessage.id}`}
+                    key={`btn-${nextMessage.id || `${index}-response-btn`}`}
                     onClick={() =>
-                      copyToClipboard(nextMessage.content, nextMessage.id)
+                      copyToClipboard(
+                        nextMessage.content,
+                        nextMessage.id || `${index}-response`
+                      )
                     }
                     className="text-xs px-2 py-1 rounded bg-gray-200 dark:bg-gray-800 text-gray-700 dark:text-gray-300 hover:bg-gray-300 dark:hover:bg-gray-700"
                     aria-label="Copy response"
                   >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)